src/pages/signin/PasswordForm.js

```diff
@@ -82,11 +83,26 @@ class PasswordForm extends React.Component {
         return (
             <>
                 <View style={[styles.mv3]}>
+                    <View
+                        style={styles.visuallyHidden}
+                        accessibilityHidden
+                    >
+                        <ExpensiTextInput
+                            value={this.props.credentials.login}
+                            autoCompleteType="username"
+                            textContentType="username"
+                            nativeID="username"
+                            name="username"
+                            autoCorrect={false}
+                        />
+                    </View>
                     <ExpensiTextInput
                         label={this.props.translate('common.password')}
                         secureTextEntry
-                        autoCompleteType="password"
+                        autoCompleteType={getPasswordAutocompleteType()}
                         textContentType="password"
+                        nativeID={getPasswordAutocompleteType()}
```

Comment:
Just curious but what do we need to set the `nativeID` for ?

Reply:
For Attaching the label with input. This could be optional. There are a lot of uncertainties in the case of password Managers so I used everything that we generally do for login forms. Another reason was to just point out these things in the review and decide if they are necessary.